x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -841,23 +856,16 @@ public void testAlias() throws Exception {
             }""");
         assertOK(adminClient().performRequest(putRoleRequest));
         // query `employees2`
-        for (String index : List.of("*:employees2", "*:employee*", "*:employee*,*:alias-employees,*:employees3")) {
+        for (String index : List.of("*:employees2", "*:employee*")) {
             Request request = esqlRequest("FROM " + index + " | KEEP emp_id | SORT emp_id | LIMIT 100");
+            System.out.println("FROM " + index);
```

Comment:
Leftover. 
(Interesting it wasn't caught by the APIs check)

Reply:
Ouch. Thanks for spotting it.